app/models/cloud_cluster.rb

```diff
@@ -66,7 +66,9 @@ def save!
     persist_to_pillar!(:cloud_worker_net, @network_id)
     persist_to_pillar!(:cloud_worker_subnet, @subnet_id)
     persist_to_pillar!(:cloud_worker_security_group, @security_group_id)
-    Velum::Salt.call(action: "saltutil.refresh_pillar")
+    Velum::Salt.call_runner(action: "saltutil.sync_pillar")
+    Velum::Salt.call(targets: "admin", action: "saltutil.refresh_pillar")
+    sleep(1)
```

Comment:
why do we need the sleep(1)? maybe add a comment

Reply:
Makes sense, I'll add one.